src/core/components/InstallButton/InstallButton.scss

```diff
@@ -220,7 +220,7 @@ $installStripeColor2: #00c42e;
   &.enabled,
   &.installed {
     input + label::before {
-      background: $switchBackgroundOn url('../img/tick.svg') no-repeat 35% 50%;
+      background: $switchBackgroundOn url('./img/tick.svg') no-repeat 35% 50%;
```

Comment:
I wonder if it's worth moving this image? This could possibly be re-used but maybe not.